### ISSUE TITLE
Make links openable in the browser

### ISSUE THREAD

**Issue by Kompwu** [mcaimi/st#7]
That'd be very handy.

**Comment by mcaimi** [comment]
Yes, I was thinking of implementing something like that a while ago. If you are interested I'll try to put that in the pipeline for future enhancements :+1:

**Comment by Kompwu** [comment]
@mcaimi Nope.

**Comment by mcaimi** [comment]
@Kompwu you mean it's not working or you do not like how it is implemented now or you did not try at all? :smile:

**Comment by Kompwu** [comment]
@mcaimi It looks so messy to me that I didn't try. :laughing:

**Comment by mcaimi** [comment]
@Kompwu :laughing: don't worry, I'll look into it
